base_uom_merge/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Base Product UOMs Merge",
+    "summary": "Merge duplicate product UOMs",
+    "version": "15.0.1.0.0",
+    "license": "LGPL-3",
```

Comment:
```suggestion
    "license": "AGPL-3",
```
based on https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade_merge_records.py

Reply:
Updated, Thanks